wikibase/1.30/bundle/Dockerfile

```diff
@@ -4,9 +4,21 @@ RUN apt-get update && \
     apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
 ADD download-extension.sh .
 RUN bash download-extension.sh OAuth
-RUN tar xzf OAuth.tar.gz && rm *.tar.gz
+RUN bash download-extension.sh Elastica
+RUN bash download-extension.sh CirrusSearch
+RUN tar xzf OAuth.tar.gz
+RUN tar xzf Elastica.tar.gz
```

Comment:
This seems slightly messy, perhaps this could all be a single RUN?

I'm aware these layers don't make it into the final image, but this will still end up using extra disk on the building machine. It might be nicer to download them, unzip them and delete the zips all in 1 step.
Or maybe even just remove the final rm *.tar.gz ? It feels a bit odd to be included only on the CirrusSearch line.

Reply:
Hm... you might be right. Obviously for a local builder it is slightly quicker to build with caching.

I think probably in this case neither speed nor disk usage matter that much and probably the most important thing is readability

Certainly the rm should go.

IMO the multiple RUNs is clearer than one long one with escaped linebreaks in the middle bit.

FWIW the current "best-practice" given that we are using multistage builds seems to be don't artificially compress multiple run lines into one: https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

However in the past, and before multistage builds it was encouraged to compress them:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Happy to hear competing thoughts though